### ISSUE TITLE
Nested Lists cannot be collapsed

### ISSUE THREAD

**Issue by Kodzhabashev** [humdek-unibe-ch/sh-selfhelp#176]
In GitLab by @moiri on Aug 14, 2019, 22:43

The style `nestedList` throws a js error when clicking on a collapse chevron:

```
ReferenceError: assignment to undeclared variable $collapsible
```
To find the code line run a similar example in debug mode.

As an example refer to the [demo](https://selfhelp.psy.unibe.ch/demo/style/362)

**Comment by Kodzhabashev** [comment]
In GitLab by @moiri on Aug 20, 2019, 14:39

closed via commit 76a4bf085a55fd6a574098690d8d64877e12fe42

**Comment by Kodzhabashev** [comment]
In GitLab by @moiri on Aug 20, 2019, 15:49

mentioned in commit 76a4bf085a55fd6a574098690d8d64877e12fe42
